### PR TITLE
Ft project setup

### PR DESCRIPTION
**Pull request by freddykaberuka**
## Completed Task

- project setup
- add linters

### CODE REVIEW

**Review by topeogunleye — COMMENTED** [review]
Hi @shasherazi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have future features, please kindly add future features to your Readme file and also fill in the other areas e.g key features.

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,249 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link]()
```
Comment (t-yanick) [inline README.md]: - [x] Great job setting up the project using React, but at the moment there's an issue in your ReadMe file.  Your demo link when clicked leads to a page with error 404. Broken links in our codebase makes it look unprofessional.  If you don't yet have a live link, you can use the text `Coming Soon`

Comment (t-yanick) [inline README.md]: - [x] Great job setting up the project using React, but at the moment there's an issue in your ReadMe file.  Your demo link when clicked leads to a page with error 404. Broken links in our codebase makes it look unprofessional.  If you don't yet have a live link, you can use the text `Coming Soon`

File: README.md [hunk 2]
```diff
@@ -1 +1,249 @@
...
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```
Comment (t-yanick) [inline README.md]: - [x] As our objective is to personalize the ReadMe as much as we can, kindly remove this line of text carry Microverse and put a more descriptive title.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @freddykaberuka , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Create React App used to create app
:heavy_check_mark: ReadMe with all sections 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Making your PR description and title more Descriptive 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @freddykaberuka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**